kubernetes/structure_persistent_volume_spec.go

```diff
@@ -1025,11 +1036,11 @@ func expandPersistentVolumeSpec(l []interface{}) (*v1.PersistentVolumeSpec, erro
 	return obj, nil
 }
 
-func expandClaimRef(v []interface{}) *v1.ObjectReference {
-	if len(v) == 0 || v[0] == nil {
-		return nil
+func expandClaimRef(l []interface{}) *v1.ObjectReference {
+	if len(l) == 0 || l[0] == nil {
+		return &v1.ObjectReference{}
 	}
-	o := v[0].(map[string]interface{})
+	o := l[0].(map[string]interface{})
 	return &v1.ObjectReference{
```

Comment:
I just changed the variable name here to be consistent with the rest of the file.